palindromic.py

```diff
@@ -1,3 +1,19 @@
-def get_longest_palindrome(s: str) -> str:
+"""This module contains function
+get_longest_palindrome(s)
+"""
 
-    return "x"
+def get_longest_palindrome(string: str) -> str:
+    """Returns a substring of a given string
+    which is a palindrome"""
+    length_s = len(string)
+
+    if length_s < 2:
+        return string
+    longest_pal = ''
+    for s_pal in range(0, length_s - 1):
```

Comment:
! You can start with the largest string, so the first palindrome found will be the longest